Expensify/App
# Fix URL rendering

### Details

1. Used Text node instead of Pressable to create inline layout inside the text to block overflow.
2. Due to on web Text does support LongPress, used `display:inline` to create inline block.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5572
$ https://github.com/Expensify/App/issues/4911

### Tests | QA Steps
1. Send a new message with a link. 
 e.g. `hello https://expensify.slack.com/archives/C01GTK53T8Q/p1632613507072000?`
 2. Check the message on native devices, does not overflow.
 3. Resize the browser on the Web, it should wrap correctly.

### Tested On


### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/143308691-e342b083-521c-4eb0-8ed5-0159818eccc6.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/143309161-6f606df0-af8b-42a9-8fb6-be53fd219f6b.mp4



#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/143308636-de9d80b4-021f-4e7c-8e35-ec7ca7b3c0c2.mp4
